apps/condo/domains/notification/constants/constants.js

```diff
@@ -116,4 +125,6 @@ module.exports = {
     MESSAGE_STATUSES,
     SHARE_TICKET_MESSAGE_TYPE,
     CUSTOMER_IMPORTANT_NOTE_TYPE,
+    PUSH_TRANSPORT_TYPES,
+    PUSH_TRANSPORT_TYPE_KEYS,
```

Comment:
Neighboring constants with such naming have a list type. It looks strange and unexpected. Usually, if the names in the same file are similar, then the types should be similar. (optional)

Reply:
fixed